src/Aspire.Hosting/Properties/Resources.resx

```diff
@@ -117,6 +117,15 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="DcpVersionCheckFailedMessage" xml:space="preserve">
+    <value>The application orchestrator version could not be determined. Continuing.</value>
+  </data>
+  <data name="DcpVersionCheckTooHighMessage" xml:space="preserve">
+    <value>The application orchestrator version is too high. Lorem ipsum how did you even get here.</value>
+  </data>
+  <data name="DcpVersionCheckTooLowMessage" xml:space="preserve">
+    <value>The application orchestrator version is too low. Run `lorem ipsum` to upgrade.</value>
```

Comment:
Need text

Reply:
How about

Newer version of .NET Aspire workload is required to run the application. Run 'dotnet workload update' to get it.